spec/03-plugins/29-acme/03-access_spec.lua

```diff
@@ -96,6 +96,29 @@ for _, strategy in helpers.each_strategy() do
 
     end)
 
+    it("dots in wildcard in domain is escaped correctly", function()
+      local res = assert( proxy_client:send {
+        method  = "GET",
+        path    = "/.well-known/acme-challenge/" .. dummy_id,
+        headers =  { host = "a.subdomain." .. do_domain }
+      })
+
+      -- key-auth should not run
+      assert.response(res).has.status(200)
+      local body = res:read_body()
+      assert.equal("isme\n", body)
```

Comment:
```suggestion
      local body = assert.response(res).has.status(200)
      assert.equal("isme", body)
```

the `status` assertion returns the body (stripped from white-space iirc)